CMakeLists.txt

```diff
@@ -273,38 +263,21 @@ if(WIN32)
             message(STATUS "Using static CRT linkage /MT")
         endif()
     else()
-        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mstackrealign -static-libgcc -static")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstackrealign -static-libstdc++ -static-libgcc -static")
-        set_target_properties(${CSOUNDLIB} PROPERTIES LINK_FLAGS "-static-libstdc++ -static-libgcc -static")
-        set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -static-libstdc++ -static-libgcc -static")
+        add_compile_options("-mstackrealign")
+        add_link_options("-static"
+            "$<$<COMPILE_LANGUAGE:C>:-static-libgcc>"
+            "$<$<COMPILE_LANGUAGE:CXX>:-static-libstdc++>"
+        )
     endif()
 endif()
 
 # MacOS/IOS c++11 flags
 if(APPLE)
-    set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -stdlib=libc++")
-    set_target_properties(${CSOUNDLIB} PROPERTIES CXX_COMPILE_FLAGS  "-std=c++11")
```

Comment:
Make sure that `target_compile_features(${CSOUNDLIB} PRIVATE cxx_std_11)` is set somewhere. I'm not entirely sure adding the `-stdlib=libc` is required. This was in the early days of C++11 when the compiler support wasn't great.

Reply:
I added `set(CMAKE_CXX_STANDARD 11)`, but will change it to be target specific. I think setting the stdlib is required for apple? https://stackoverflow.com/questions/14228856/how-to-compile-c-with-c11-support-in-mac-terminal

Reply:
That's from 2013. That shouldn't be needed anymore